WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -1775,9 +1778,21 @@ public static void showLoginPrologue(@NonNull Context context) {
         context.startActivity(intent);
     }
 
-    public static void startJetpackMigrationFlow(@NonNull Context context) {
+    public static void startJetpackMigrationFlow(@NonNull Context context,
+                                                 @Nullable PreMigrationDeepLinkData deepLinkData) {
         Intent intent = new Intent(context, JetpackMigrationActivity.class);
         intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK | Intent.FLAG_ACTIVITY_NEW_TASK);
+        if (deepLinkData != null) {
+            intent.putExtra(KEY_DEEP_LINK_DATA, deepLinkData);
+        }
+        context.startActivity(intent);
+    }
+
+    public static void openDeepLinkAfterJPMigration(@NonNull Context context, String action, Uri uri) {
+        Intent intent = new Intent(context, DeepLinkingIntentReceiverActivity.class);
+        intent.setAction(action);
+        intent.setData(uri);
+        intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK | Intent.FLAG_ACTIVITY_NEW_TASK);
         context.startActivity(intent);
```

Comment:
@zwarm I added you as reviewer (since you're the author of the latest deep links changes) to please verify if you don't foresee any possible issues with running this code at the end of the migration flow in order to open the app at the screen targeted in the deep links 🙏 🙇‍♂️

Reply:
The code was changed after the implementation to trigger migration from post web links, this is no longer a concern because the Jetpack app is [set explicitly to handle the link after migration](https://github.com/wordpress-mobile/WordPress-Android/pull/17742/files#diff-200f30b456241b2103300badbb3b04bfe696ce8386f45a8781c7d95db564c677R1790-R1795).